File: Fortran and C Codes by Salvi/Fortran Codes/Lab Problems/fibonacci.f90
PROGRAM FIBONACCI
    INTEGER T1,T2
    PRINT*,"HOW MANY TERMS OF FIBONACCI SERIES?"
    READ*,N
    T1=0
    T2=1
    PRINT*,"THE FIBONACCI SERIES OF ",N," TERMS IS"
    PRINT*,T1
    PRINT*,T2
    I=3
    10  NT=T1+T2
        T1=T2
        T2=NT
        PRINT*,T2
        I=I+1
        IF(I.LE.N)GOTO 10
    STOP
END PROGRAM

File: Fortran and C Codes by Salvi/C Codes/Random Problems/fibonacci.c
#include <stdio.h>

int main() {
    int n;

    printf("Enter the number of terms for Fibonacci series: ");
    scanf("%d", &n);

    int t1 = 0, t2 = 1;

    printf("Fibonacci series:\n");

    for (int i = 1; i <= n; i++) {
        printf("%d ", t1);

        int nt = t1 + t2;
        t1 = t2;
        t2 = nt;
    }

    printf("\n");

    return 0;
}
